src/Widget/Attribute/CommonAttributes.php

```diff
@@ -71,54 +34,62 @@ public function autofocus(bool $value = true): self
      *
      * @param bool $value
      *
-     * @return self
+     * @return static
+     *
+     * @link https://www.w3.org/TR/html52/sec-forms.html#element-attrdef-disabledformelements-disabled
      */
     public function disabled(bool $value = true): self
     {
         $new = clone $this;
-        $new->options['disabled'] = $value;
+        $new->attributes['disabled'] = $value;
         return $new;
     }
 
     /**
-     * The earliest acceptable date.
+     * Specifies the form element the tag input element belongs to. The value of this attribute must be the id
+     * attribute of a {@see Form} element in the same document.
      *
      * @param string $value
      *
-     * @return self
+     * @return static
+     *
+     * @link https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fae-form
      */
-    public function min(string $value): self
+    public function form(string $value): self
     {
         $new = clone $this;
-        $new->options['min'] = $value;
+        $new->attributes['form'] = $value;
         return $new;
     }
 
     /**
-     * The latest acceptable date.
+     * If it is required to fill in a value in order to submit the form.
      *
-     * @param string $value
+     * @param bool $value
      *
-     * @return self
+     * @return static
+     *
+     * @link https://www.w3.org/TR/html52/sec-forms.html#the-required-attribute
      */
-    public function max(string $value): self
+    public function required(bool $value = true): self
     {
         $new = clone $this;
-        $new->options['max'] = $value;
+        $new->attributes['required'] = true;
```

Comment:
```suggestion
        $new->attributes['required'] = $value;
```

or remove an argument.

Reply:
done.